src/Nav.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types'
+import Container from 'react-bootstrap/Container'
+import BNav from 'react-bootstrap/Nav'
```

Comment:
When using a default import, you can name it whatever you want.

React components, per convention, always start with capital letter. To avoid name overlapping, I have named the Bootstrap Nav as `BNav` and your `_nav` can be `Nav`.